pkg/action/install.go

```diff
@@ -106,14 +106,23 @@ type Install struct {
 	Description      string
 	OutputDir        string
 	// RollbackOnFailure enables rolling back (uninstalling) the release on failure if set
-	RollbackOnFailure        bool
+	RollbackOnFailure bool
+	// SkipCRDs opts the chart installation out of installing or upgrading CRs on the cluster. By
+	// default, CRDs are installed (or upgraded) during a chart installation and AlreadyExists errors
+	// are ignored. When this flag is set, no operations over CRDs will be made against the cluster
+	// during installation.
```

Comment:
```suggestion
	// SkipCRDs If set to false, CRDs are installed (or upgraded) during a chart installation and
	// AlreadyExists errors are ignored. If set to true, no operations on CRDs are performed on
	// the cluster during installation.
```

Reply:
>SkipCRDs If set to false

Not really how GoDoc is normally written - adapted your comment to make it flow a little nicer.